Userland/Applications/DisplaySettings/FontSettingsWidget.cpp

```diff
@@ -18,40 +18,44 @@ FontSettingsWidget::FontSettingsWidget()
 {
     load_from_gml(font_settings_gml);
 
-    auto& default_font_label = *find_descendant_of_type_named<GUI::Label>("default_font_label");
-    auto& default_font_button = *find_descendant_of_type_named<GUI::Button>("default_font_button");
-    auto& fixed_width_font_label = *find_descendant_of_type_named<GUI::Label>("fixed_width_font_label");
-    auto& fixed_width_font_button = *find_descendant_of_type_named<GUI::Button>("fixed_width_font_button");
-
-    default_font_label.set_font(Gfx::FontDatabase::default_font());
-    fixed_width_font_label.set_font(Gfx::FontDatabase::default_fixed_width_font());
+    auto& default_font = Gfx::FontDatabase::default_font();
+    m_default_font_label = *find_descendant_of_type_named<GUI::Label>("default_font_label");
+    update_label_with_font(m_default_font_label, default_font);
 
-    default_font_button.on_click = [this, &default_font_label] {
-        auto font_picker = GUI::FontPicker::construct(window(), &default_font_label.font(), false);
+    auto& default_font_button = *find_descendant_of_type_named<GUI::Button>("default_font_button");
+    default_font_button.on_click = [this] {
+        auto font_picker = GUI::FontPicker::construct(window(), &m_default_font_label->font(), false);
         if (font_picker->exec() == GUI::Dialog::ExecOK) {
-            default_font_label.set_font(font_picker->font());
-            default_font_label.set_text(font_picker->font()->qualified_name());
+            update_label_with_font(m_default_font_label, font_picker->font());
         }
     };
 
-    fixed_width_font_button.on_click = [this, &fixed_width_font_label] {
-        auto font_picker = GUI::FontPicker::construct(window(), &fixed_width_font_label.font(), true);
+    auto& default_fixed_width_font = Gfx::FontDatabase::default_fixed_width_font();
+    m_fixed_width_font_label = *find_descendant_of_type_named<GUI::Label>("fixed_width_font_label");
+    update_label_with_font(m_fixed_width_font_label, default_fixed_width_font);
+
+    auto& fixed_width_font_button = *find_descendant_of_type_named<GUI::Button>("fixed_width_font_button");
+    fixed_width_font_button.on_click = [this] {
+        auto font_picker = GUI::FontPicker::construct(window(), &m_fixed_width_font_label->font(), true);
         if (font_picker->exec() == GUI::Dialog::ExecOK) {
-            fixed_width_font_label.set_font(font_picker->font());
-            fixed_width_font_label.set_text(font_picker->font()->qualified_name());
+            update_label_with_font(m_fixed_width_font_label, font_picker->font());
         }
     };
 }
 
-FontSettingsWidget::~FontSettingsWidget()
+void FontSettingsWidget::update_label_with_font(RefPtr<GUI::Label> label, RefPtr<Gfx::Font> const& font)
 {
+    label->set_text(font->qualified_name());
+    label->set_font(font);
```

Comment:
you can just take a `GUI::Label&` and `Gfx::Font const&` here, no need for the (nullable) refptr.

Reply:
Ah yes, this was also my original intention. I've only now found out how to do that... I think! I'll send in my patches soon, hope they are OK.

Reply:
As I am a bit unsure how this reviewing process in Serenity works - am I supposed to "resolve" then I think I've done the change? Because that's how I know it. Because of that, I'll do that here and you can re-open if I was in the wrong.

Reply:
yeah please do that if you think a comment is resolved! makes it easier to see what's left to do or if a re-review can be done :)

Reply:
That's good to know! I've seen the linter error... Now it should work.